### DISCUSSION TITLE
Sonoff DUAL

### DISCUSSION THREAD

**Discussion by ViktorValnicek** [RavenSystem/esp-homekit-devices#2002] [MEPLHAA Scripts Collection]
Hi, could you please convert my sonoff dual config to v12..

my old working config:

{"c":{
"l":13,"b":[{"g":10,"t":5}]},
"a":[{"0":{"r":[{"g":12}]},
"1":{"r":[{"g":12,"v":1}]},
"b":[{"g":0,"t":0}]},
{"0":{"r":[{"g":5}]},
"1":{"r":[{"g":5,"v":1}]},
"b":[{"g":9,"t":0}]}]}

thanks

**Comment by xrust83** [comment]
{"c":{"io":[[[13],3],[[0,9,10],6],[[5,12],2]],
"l":13,"b":[{"g":10,"t":5}]},
"a":[{"0":{"r":[{"g":12}]},
"1":{"r":[{"g":12,"v":1}]},
"b":[{"g":0,"t":0}]},
{"0":{"r":[{"g":5}]},
"1":{"r":[{"g":5,"v":1}]},
"b":[{"g":9,"t":0}]}]}
